### ISSUE TITLE
Failed to configure target

### ISSUE THREAD

**Issue by rgaufman** [osxfuse/osxfuse#605]
I am following the installation instructions, but seeing this:

```
./build.sh -t distribution
T:distribution       | Clean target
T:distribution       | Build target for macOS 10.14
T:fsbundle           | Clean target
T:fsbundle           | Build target for macOS 10.14
T:kext               | Clean target
T:kext               | Build target for macOS 10.14
T:kext               | Install target
T:fsbundle           | Install target
T:library            | Clean target
T:library            | Build target for macOS 10.14
T:library            | Failed to configure target
Terminated: 15
```

Any ideas?

**Comment by bfleischer** [comment]
Did you install all dependencies listed in the README? Please run `./build.sh -v 5 -t distribution` to increase the verbosity of the build script.

**Comment by rgaufman** [comment]
I installed the dependencies and seeing this:

```
$ ./build.sh -v 5 -t distribution
DEFAULT_BUILD_CONFIGURATION=Release
DEFAULT_BUILD_DIRECTORY=/tmp/osxfuse
DEFAULT_LOG_VERBOSE=2
DEFAULT_PREFIX=''
DEFAULT_SDK=10.14
DEFAULT_SDK_10_10_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_10_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_11_ARCHITECURES=([0]=i386 [1]=x86_64)
T:library            | BUILD_TARGET_OPTION_DEBUG_DIRECTORY=''
T:library            | BUILD_TARGET_OPTION_DEPLOYMENT_TARGET=10.14
T:library            | BUILD_TARGET_OPTION_GROUP=''
T:library            | BUILD_TARGET_OPTION_MACROS=()
T:library            | BUILD_TARGET_OPTION_OWNER=''
T:library            | BUILD_TARGET_OPTION_PREFIX=/usr/local
T:library            | BUILD_TARGET_OPTION_PRODUCT_SIGN_IDENTITY=''
T:library            | BUILD_TARGET_OPTION_PROVISIONING_PROFILE_SPECIFIER=''
T:library            | BUILD_TARGET_OPTION_ROOT=0
T:library            | BUILD_TARGET_OPTION_SDK=10.14
T:library            | BUILD_TARGET_OPTION_XCODE=10.2.1
T:library            | Removing target build directory '/tmp/osxfuse/library'
T:library            | Completed action clean
T:library            | Build target for macOS 10.14
Running libtoolize...
glibtoolize: putting auxiliary files in '.'.
glibtoolize: copying file './ltmain.sh'
glibtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
glibtoolize: copying file 'm4/libtool.m4'
glibtoolize: copying file 'm4/ltoptions.m4'
glibtoolize: copying file 'm4/ltsugar.m4'
glibtoolize: copying file 'm4/ltversion.m4'
glibtoolize: copying file 'm4/lt~obsolete.m4'
Running autoreconf...
configure.ac:10: installing './compile'
configure.ac:5: installing './config.guess'
configure.ac:5: installing './config.sub'
configure.ac:6: installing './install-sh'
configure.ac:6: installing './missing'
example/Makefile.am: installing './depcomp'
checking build system type... x86_64-apple-darwin18.6.0
checking host system type... x86_64-apple-darwin18.6.0
checking target system type... x86_64-apple-darwin18.6.0
checking for a BSD-compatible install... /usr/local/opt/coreutils/libexec/gnubin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/local/opt/coreutils/libexec/gnubin/mkdir -p
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether /Applications/Xcode.app/Contents/Developer/usr/bin/make sets $(MAKE)... yes
checking whether /Applications/Xcode.app/Contents/Developer/usr/bin/make supports nested variables... yes
checking whether /Applications/Xcode.app/Contents/Developer/usr/bin/make supports nested variables... (cached) yes
checking how to print strings... printf
checking whether /Applications/Xcode.app/Contents/Developer/usr/bin/make supports the include directive... yes (GNU style)
checking for gcc... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
checking whether the C compiler works... no
configure: error: in `/tmp/osxfuse/library/Source':
configure: error: C compiler cannot create executables
See `config.log' for more details
T:library            | Failed to configure target
Terminated: 15
Received signal: SIGTERM
$
```

**Comment by bfleischer** [comment]
Have you checked the config.log file? This is most likely an issue with your build environment.

May I ask why you are trying to build FUSE yourself? It would be so much easier to use the official installer.

**Comment by rgaufman** [comment]
I thought I'm meant to :) - brew doctor isn't showing any errors and I don't have any issues compiling anything else. I am trying to compile because I started receiving this error:

<img width="532" alt="Screenshot 2019-07-19 at 23 30 37" src="https://user-images.githubusercontent.com/33842342/61569049-6e51cf80-aa7d-11e9-9e74-d5d2137ec503.png">

I've used the official installer and my problem went away! - Thank you!
